docs/developers/setup.md

```diff
@@ -68,13 +68,13 @@ these networks, you'll generally need to know:
 2. What private key you want to use and how your private keys are stored.
    Below is information regarding the two types of keys that are supported, mnemonics/seed phrases and hardware wallets.
 
-The `--network` parameter that's passed to all truffle commands depends on both of these factors. Here's an example:
+The `--network` parameter that's passed to all hardhat commands depends on both of these factors. Here's an example:
 
 ```bash
-yarn truffle console --network rinkeby_mnemonic
+npx hardhat console --network rinkeby_mnemonic
 ```
 
-That command will tell truffle that the user wants to use the Rinkeby testnet and their private key is a mnemonic, or
+That command will tell hardhat that the user wants to use the Rinkeby testnet and their private key is a mnemonic, or
```

Comment:
I think we need to take out the private key and mnemonic part of this sentence and change the network argument to be structured as `--network [NETWORK_NAME]`